src/graphics/opengl/RendererGL.cpp

```diff
@@ -567,7 +570,8 @@ namespace Graphics {
 
 	bool RendererOGL::SetDepthRange(double znear, double zfar)
 	{
-		glDepthRange(znear, zfar);
+		// XXX since we're using reverse-Z, flip the inputs to this function to avoid breaking old code.
+		glDepthRange(1.0 - zfar, 1.0 - znear);
```

Comment:
ooh, I'm torn about this decision. Do we just want to make it explicit, possibly by renaming the method and parameters? Tricky. Is it commonly used by non-graphics code? Because if it's just people who are more familiar with the graphics stuff, and know that it's using reverse-Z, then they might double reverse it?

Reply:
I'd like to totally remove the DepthRange call (or at least make it part of a pipeline state object) as it's a bit abused by the SectorMapView at the moment.